src/main/java/com/savvato/tribeapp/services/UserRoleMapServiceImpl.java

```diff
@@ -7,56 +7,53 @@
 import com.savvato.tribeapp.repositories.UserRoleMapRepository;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Service
 public class UserRoleMapServiceImpl implements UserRoleMapService {
-	@Autowired
-	UserRoleMapRepository userRoleMapRepo;
-
-	public List<String> getRoles() {
-		ArrayList<String> rtn = new ArrayList<>();
-
-		for (ROLES role : ROLES.values()) {
-			rtn.add(role.name());
-		}
-
-		return rtn;
-	}
```

Comment:
This is the only change that was made to the UserRoleMapServiceImpl file. I'm continuing to explore why certain files seem to randomly get formatted differently, I'm really sorry about the inconvenience!